job-server/src/main/scala/spark/jobserver/JobManagerActor.scala

```diff
@@ -316,6 +315,7 @@ class JobManagerActor(contextConfig: Config, daoActor: ActorRef) extends Instrum
         // Either way an enhancement would be required here to make Stream[_] responses work
         // with context-per-jvm=true configuration
         resultActor ! JobResult(jobId, result)
+        statusActor ! JobFinished(jobId, DateTime.now())
```

Comment:
I'm fine with this, but just to note that the JobResultActor should really be local to the driver JVM, which would be a better fix.